CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing
+
+Welcome, and thank you for your interest in contributing to the AL developer tools! The goal of this document is to provide a high-level overview of how you can get involved.
+
+## Asking questions
+
+Have a question? Rather than opening an issue, please use one of the following resources:
+
+- [Business Central Community](https://community.dynamics.com/business/f/758)  
+- [mibuso forum](https://forum.mibuso.com/categories/nav-three-tier)  
+- [Dynamics User Group](https://dynamicsuser.net/nav/f/developers)  
+  
+The community will be eager to assist you and your well-worded question will serve as a resource to others searching for help.
+
+## Reporting issues
+
+Have you identified a reproducible problem in the AL compiler or developer tools? Here's how you can make reporting your issue as effective as possible.
+
+### Identify where to report
+
+The Dynamics 365 Business Central team accepts feedback through multiple channels and repositories. Try to file the issue through the correct channel to ensure that your issue is addressed effectively and efficiently.
+
+- For issues related to bugs in the latest version of the AL Language extension for Visual Studio Code or any of the static analysis tools that accompany the AL Language extension, use the current [GitHub repository](https://github.com/microsoft/al). You should always use the latest version of the AL Language extension when targeting any version of Business Central. If you are targeting Dynamics NAV 2018 or older, please use the [support channel](#support).
+- For new features in AL/Visual Studio Code language and tools, as well as Business Central in general, use [Business Central Ideas](https://aka.ms/bcideas). This will allow other members of the community to vote on your idea and will allow us to prioritize based accordingly.
+- For suggestions for new static analysis rules or improvements to existing static analysis rules, use [Business Central Ideas](https://aka.ms/bcideas).
+- For issues related to extensions authored by Microsoft, such as the `System Application` or the `Base Application` as well as event requests, use the [ALAppExtensions](https://github.com/Microsoft/ALAppExtensions) repository.
+- For issues concerning versions of Business Central that are in mainstream support you should use the [support channel](#support).
+
+### Look for an existing issue
+
+Before you create a new issue, please do a search in [open issues](https://github.com/microsoft/AL/issues) to see if the issue or feature request has already been filed.
```

Comment:
```suggestion
Before you create a new issue, please do a search in [open issues](https://github.com/microsoft/AL/issues) to see if the issue or feature request has already been filed, and add to that instead.
```

Reply:
As long as issues often get closed with "won't fix", it would be helpful to search also in closed issues. Unless you want us to create new issues for the same questions and problems over and over again.